TCAT/Views/Circle.swift

```diff
@@ -26,7 +26,7 @@ class Circle: UIView {
 
     // MARK: Data vars
 
-    let diameter: CGFloat
+    private let diameter: CGFloat
```

Comment:
Do we actually use the value of `diameter` directly anywhere, or is always `diameter / 2`? In the latter case, we should have a `radius` variable instead.